Sources/Common/Core/MatrixBuilder/index.d.ts

```diff
@@ -24,7 +24,7 @@ declare interface Transform {
 	 * @param {Number} angle 
 	 * @param {Number} axis 
 	 */
-	rotate(angle: number, axis: number): Transform
+	rotate(angle: number, axis: number[]): Transform
```

Comment:
you can use the types from https://github.com/Kitware/vtk-js/blob/master/Sources/types.d.ts#L20-L22